src/Word/SeparatorToCamelCase.php

```diff
@@ -4,65 +4,58 @@
 
 namespace Laminas\Filter\Word;
 
-use Closure;
-use Laminas\Stdlib\StringUtils;
+use Laminas\Filter\FilterInterface;
+use Laminas\Filter\ScalarOrArrayFilterCallback;
 
 use function mb_strtoupper;
 use function preg_quote;
 use function preg_replace_callback;
-use function strtoupper;
 
 /**
  * @psalm-type Options = array{
  *     separator?: string,
- *     ...
  * }
  * @template TOptions of Options
```

Comment:
This template is no longer necessary

Reply:
Thanks for taking a look. I've amended that